src/common/lcf_widget_binding.h

```diff
@@ -121,7 +121,7 @@ namespace LcfWidgetBinding {
 			}
 
 			auto data = variant.value<LcfObjectHolder<T>>();
-			data.obj() = static_cast<T>(comboBox->currentIndex());
+			data.obj() = static_cast<T>(comboBox->currentData().toInt());
```

Comment:
bet this will break something but that change makes sense

Reply:
Since the same change was inplemented in #200, this PR no longer contains this because of the rebase.